pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -185,9 +195,32 @@ func (vm *windows) Configure() error {
 	if err := vm.transferFiles(); err != nil {
 		return errors.Wrap(err, "error transferring files to Windows VM")
 	}
+	if err := vm.ConfigureWindowsExporter(); err != nil {
```

Comment:
I am assuming there is no issue running this before the node is prepped

Reply:
Correct